context-propagation/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+quarkus.datasource.url=jdbc:postgresql://localhost/quarkus_test
```

Comment:
I could be wrong, but doesn't this need mp config for the reactive connector type?

Reply:
no, here we want to use the blocking one.